README.md

```diff
@@ -128,6 +128,30 @@ if (!supportsTime) {
 
 This will add a global `TimePolyfill` function to the page.
 
+### Shadow DOM
+
+When your code is inside a component that resides in the Shadow DOM, the polyfill may not be able to find your label
+element. For this case, you can pass your label element in directly.
+
+```
+<label id="myLabel"></label>
+<input #time type="time" id="timeInput">
+```         
+
+```               
+// This example is in Angular/Typscript
```

Comment:
Don't use angular or Typescript for the example. Anyone using those things will know how to translate the example. People who don't use those things will just get confused by the Typescript and Angular parts.